atf_metrics/src/atf_metrics/calculate_resources.py

```diff
@@ -102,22 +123,43 @@ def process_resource_data(self, msg):
                     pass
 
     def get_result(self):
+        groundtruth_result = None
+        average_sum = 0.0
+
         if self.finished:
+            #print "----------------------------- \n node data:", self.node_data
             for node in self.node_data:
+                #print " node:", node
                 for res in self.node_data[node]:
+                    #print "res", res
                     if len(self.node_data[node][res]["data"]) != 0:
                         if res == "io" or res == "network":
                             for values in self.node_data[node][res]["data"]:
                                 self.node_data[node][res]["average"].append(float(round(numpy.mean(values), 2)))
                                 self.node_data[node][res]["min"].append(float(round(min(values), 2)))
                                 self.node_data[node][res]["max"].append(float(round(max(values), 2)))
+                                average_sum += float(round(numpy.mean(values), 2))
+                                #print "average sum:", average_sum
                         else:
                             self.node_data[node][res]["average"] = float(round(numpy.mean(self.node_data[node][res]
                                                                                           ["data"]), 2))
                             self.node_data[node][res]["min"] = float(round(min(self.node_data[node][res]["data"]), 2))
                             self.node_data[node][res]["max"] = float(round(max(self.node_data[node][res]["data"]), 2))
+                            average_sum += float(round(numpy.mean(self.node_data[node][res]["data"]), 2))
+                            #print "average sum:", average_sum
                     del self.node_data[node][res]["data"]
 
-            return "resources", self.node_data
+                #print "groundtruthes:", self.groundtruth, self.groundtruth_epsilon, "\n average:", self.node_data[node][res]["average"]
+                if self.groundtruth != None and self.groundtruth_epsilon != None:
+                    for node in self.node_data:
+
+                        #print "average sum:check", average_sum
+                        if math.fabs(self.groundtruth - average_sum) <= self.groundtruth_epsilon:
+                            groundtruth_result = True
+                        else:
+                            groundtruth_result = False
+
+            #print "node data: ", self.node_data, "\n groundthruth result", groundtruth_result, " \n .................................."
+            return "resources", self.node_data, groundtruth_result, self.groundtruth, self.groundtruth_epsilon, "*resources details*"
```

Comment:
let's use `details = None` if there is no usefull things to wrap in here. see https://github.com/ipa-fmw/atf/blob/master/atf_metrics/src/atf_metrics/calculate_time.py#L68